src/circom/builder.rs

```diff
@@ -27,6 +27,7 @@ impl<E: PairingEngine> CircomConfig<E> {
     pub fn new(wtns: impl AsRef<Path>, r1cs: impl AsRef<Path>) -> Result<Self> {
         let wtns = WitnessCalculator::new(wtns).unwrap();
         let reader = File::open(r1cs)?;
+        println!("CircomConfig new, about to read new R1CS file");
```

Comment:
Small recommendation, if you're adding logging anyway, it's probably better to use [tracing](https://docs.rs/tracing) with [fmt subscriber](https://docs.rs/tracing-subscriber/0.3.2/tracing_subscriber/fmt/index.html)

Reply:
Good idea! I mostly did it for my own debugging. I suggest we introduce this in a separate PR as it is orthogonal to current PR.